includes/adapter/RecipientFromWCOrder.php

```diff
@@ -90,17 +90,17 @@ private function getAddressFromOrder(WC_Order $order, string $type): array
 
         $addressLine2IsNumberSuffix = strlen($addressLine2) < self::MIN_STREET_ADDITIONAL_INFO_LENGTH;
 
-        if (! $streetParts['number_suffix'] && $addressLine2IsNumberSuffix) {
+        if (isset($streetParts['number_suffix']) && $addressLine2IsNumberSuffix) {
```

Comment:
Dit ziet er raar uit voor mij nu: wanneer number_suffix bestaat in streetParts moet het overschreven worden? En als het niet bestaat, wordt het dus niet overschreven ook niet als addresLine2IsNumberSuffix? Als het klopt heb ik niets gezegd natuurlijk.